gossip/src/crds.rs

```diff
@@ -669,6 +686,18 @@ impl CrdsDataStats {
                 self.votes.put(slot, num_nodes + 1);
             }
         }
+
+        if should_report_message_signature(entry.value.signature) {
+            datapoint_info!(
+                "cluster_info_crds_message_signatures",
+                ("crds_origin", entry.value.pubkey().to_string(), String),
+                (
+                    "crds_signature",
+                    bs58::encode(&entry.value.signature.as_ref()[..8]).into_string(),
```

Comment:
Why not `entry.value.signature.to_string().get(..8)` here?

Reply:
so this will take the first 8 characters that are bs58 encoded right? which is technically less than 8 bytes right?

Reply:
also `entry.value.signature.to_string().get(..8)` returns an &Option<&str>. what's the best way to avoid using an `unwrap()` here to extract the string?

Reply:
I don't think it matters much. We just need an almost unique identifier, so I think 8 bytes should be enough.

Reply:
makes sense!